tests/test_on_data/test_behavior_interfaces.py

```diff
@@ -480,7 +480,7 @@ def check_metadata(self):
         assert metadata["NWBFile"]["session_start_time"] == datetime(2023, 5, 15, 10, 35, 29)
 
 
-class TestSLEAPInterface(TestCase):
+class CustomTestSLEAPInterface(TestCase):
```

Comment:
there is another class in this file named TestSLEAPInterface

Reply:
This one looks to be the older style before the new mixin was added (and when mixin was added, this one wasn't removed)

I think the goal should be to remove this one in a follow up and absorb any differences with its test function into the new mixin (if there are any differences that is) - probably in a follow-up

Reply:
Yeah, I thought about deleting it but it looked like there were some differences and I didn't know exactly how to merge them. I agree, there is probably a lot of redundancy here.